autogpt_platform/backend/backend/blocks/codex.py

```
@@ -152,16 +221,23 @@ def __init__(self):
         self.execution_stats = NodeExecutionStats()
 
-    # GPT-5.1-Codex published pricing: $1.25 / 1M input, $10 / 1M output.
-    _INPUT_USD_PER_1M = 1.25
-    _OUTPUT_USD_PER_1M = 10.0
+    _MODEL_USD_PER_1M: dict[CodexModel, tuple[float, float]] = {
+        CodexModel.GPT5_6_SOL: (5.0, 30.0),
+        CodexModel.GPT5_6_TERRA: (2.5, 15.0),
+        CodexModel.GPT5_6_LUNA: (1.0, 6.0),
+        CodexModel.GPT5_3_CODEX: (1.75, 14.0),
+        CodexModel.GPT5_1_CODEX: (1.25, 10.0),
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)